### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by davidanthoff** [uwdata/lyra#412]
Just a quick question: is this project still being maintained? Any future plans? It looks really cool, but there doesn't seem to be too much activity in the repo?

**Comment by jheer** [comment]
There is no current activity on the project, but I believe it may resume in the summer or fall. Paging @arvind who can provide a more definitive answer.

**Comment by davidanthoff** [comment]
It would be great if it would resume! I am in awe of all the stuff you guys created, and feel one could create some really powerful integrations for julia. I've sketched some ideas in https://github.com/vega/voyager/issues/745#issuecomment-353530784 and https://github.com/vega/vega-lite/issues/1472#issuecomment-353682418.

**Comment by TomBor** [comment]
Yes, It would be great to know if there is plan (or not) to continue Lyra.

**Comment by arvind** [comment]
Hi folks, thanks for the continued interest (and apologies for the delayed response). Yes, we expect to resume work on Lyra this summer. 

We were frustratingly close to a Lyra 2.0 release back in 2016 but, unfortunately, needed to reprioritize our efforts towards, among other things, releasing Vega-Lite 2.0.

The good news is that work will not go to waste — it laid a very solid foundation that will make rebuilding momentum behind this project much easier. Our primary task will be to update Lyra’s model to Vega 3/Vega-Lite 2, which should be a fairly straightforward process (aides, hopefully, by migrating the codebase to TypeScript). And once we do, we can begin to build on it much more quickly than before.

**Comment by TomBor** [comment]
Great news! I'm looking forward to test it.
Thanks for the update.

**Comment by davidanthoff** [comment]
Excellent!

**Comment by davidanthoff** [comment]
Any update what the current plan is?

**Comment by arvind** [comment]
Thanks for checking in @davidanthoff! Returning to Lyra is high on our priority list but, unfortunately, we don’t have any updates yet.

**Comment by davidanthoff** [comment]
Cool, thanks for the update!

**Comment by Sigfried** [comment]
Hi. I'm here wondering the same thing.

I decided to use Vega for an experimental network data exploration application and I'd love to be able to start with the basic framework already in place. I don't know if Lyra would have worked for that anyway, but if it's not going to integrate with current versions of Vega/Vega-Lite (I'll probably need to use Vega mostly), it's probably not a good match.

Thanks for all your amazing work!

**Comment by arvind** [comment]
Good timing @sigfried. We’ve just restarted work on Lyra. We don’t have a timeline yet for when work should be complete porting it over to the latest versions of Vega and Vega-Life, but doing so is one of the highest priority items on our agenda!

**Comment by dprophet** [comment]
Hi @arvind Can you update the build dependencies and build instructions for the lyra2 branch? I am interested in seeing Lyra move forward. Lyra is the closest there is to an open source Tableau.

**Comment by jonathanzong** [comment]
Hi @dprophet,

There are two working branches for Lyra 2 right now. `lyra2` is the most recently working version that is usable, but it does not yet have all functionality implemented. If you're just looking to test something out, that's the right branch. `lyra2019` is being actively developed, but will be unstable in the short term (next few months). If you're interested in digging into the latest code, this is the right branch.

I've checked the README on both branches to make sure they're up to date, let us know if you run into issues or are looking for some specific piece of info. Thanks!
